pyclibrary/headers/update_WinDefs.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015 by PyCLibrary Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the MIT/X11 license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""
+Windows headers are not included in this distribution due to possible
+copyright issues. Thus there is the cache file 'WinDefs.cache' here
+which encapsulate definitions pulled from several headers included
+in Visual Studio
+
+This script updatthe cache fileche.
+It has to be run on one of the following conditions:
+* a another Visual C/C++ compiler version is used
+* the object model of the parser was updated.
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+from pyclibrary.c_parser import win_defs
+from pyclibrary.utils import add_header_locations
+
+SDK_DIR = r'c:\program files\microsoft sdks\windows\v6.0a\include'
```

Comment:
Could we use a regex or something smarter here, to detect the correct path ? (just wondering you don't have to do it now)